Changelog.md

```diff
@@ -27,6 +27,9 @@ Features:
  * Syntax Checker: Issue error if no visibility is specified on contract functions as experimental 0.5.0 feature.
  * Syntax Checker: Issue warning when using overloads of ``address`` on contract instances.
  * Type Checker: disallow combining hex numbers and unit denominations as experimental 0.5.0 feature.
+ * Improved messaging when error spans multiple lines of a sourcefile 
+ * Commandline interface:  Error when missing file detected. Support to suppress with missing error ``--ignore-missing`` flag.
```

Comment:
Please move this to the 0.4.22 section.

Reply:
@chriseth Yup, my apologies.  I'll have an updated and squashed (removing merge) tonight.